### ISSUE TITLE
Octoprint/web interface folders

### ISSUE THREAD

**Issue by eviglotti** [Robo3D/OctoPrint-robotheme#16]
Hello, per this url, this appears to be a bug with the Robo skin. How do I remove the Robo skin and then do I then lose functionality such as the webcam, stats, etc?

http://community.robo3d.com/index.php?threads/octoprint-web-interface-folders.19107/

Thanks!

**Comment by AllenMcAfee** [comment]
@eviglotti can you describe the bug you are experiencing?  

Disabling the theme is easy.  Under Settings/Plugins you will look for the robotheme under page 4 of the plugin list.  From there you can click the black circle on the right of the name to disable the plugin.  From here you will need to reset Octoprint for the changes to take hold.

Let us know if you have any more questions!

**Comment by eviglotti** [comment]
Perfect, I will try that and see how it goes. What's interesting is that when I created two folders in Octoprint, they wouldn't show up no matter how many times I rebooted my browser. I then went to the printer and used the LCD screen to move a few files into one of the two folders leaving the other one empty. Now in Octoprint, I see the folder that has files, but still don't see the empty folder. So maybe this is a bug/feature that Octoprint is only showing folders that have files in them.

**Comment by AllenMcAfee** [comment]
I believe this is an issue with Octoprint.  We are rebuilding the file system for the LCD in the next release and we will investigate this issue.  Thank you for bringing this to our attention!

**Comment by Ximidar** [comment]
This is actually a setting in octoprint. Under the files there is a wrench icon that had an option to show folders with nothing inside them. This is not shown with robo theme, but it is shown with the default theme

**Comment by eviglotti** [comment]
Interesting, in the meantime I think I'll remove the robo skin and see how it goes. Thanks!

**Comment by eviglotti** [comment]
Very good info Ximidar. I was able to disable the Robo skin (which then required me to physically power off the printer as it would not reboot and come up without doing that), then I saw the wrench where I could disable the option you listed. I now saw all folders, empty or not. Then I was able to re-activate the Robo skin and now I have both the robo skin AND I can see empty folders. A bit complex, but I'm good to go, so thanks for the comments everyone!
